### PR TITLE
Added sbt-native-packager settings to package the 'sigar' directory and create 2 zips

### PR DESCRIPTION
**Pull request by UriShohet**
Hi @muuki88 !

First of all I'd like to thank you for the great play+akka cluster tutorial!

I'd like to propose a change to your code to create 2 zip packages, one for backend and one for frontend, both having the sigar folder inside. When I ran the commands as described in the tutorial all worked great, but when I ran 'activator dist' - only frontend zip was created and when I executed the script it failed complaining about missing sigar .so in the java.library.path. Also, there was no way to execute the backend.

I'm only a beginner in play/sbt/github stuff, so please be gentle ;-)

Please tell me what you think.

Thanks in advance,
Uri Shohet

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -19,7 +22,7 @@ lazy val root = (project in file("."))
   .aggregate(api, frontend, backend)
   
 lazy val frontend = (project in file("frontend"))
-    .enablePlugins(PlayScala, BuildInfoPlugin)
+    .enablePlugins(PlayScala, BuildInfoPlugin, JavaAppPackaging)
```
Comment (muuki88) [inline build.sbt]: I think this is not really needed, as play already adds the server_archetype. However we can keep this to make things more clear.

File: build.sbt [hunk 2]
```diff
@@ -42,6 +50,11 @@ lazy val backend = (project in file("backend"))
             "-Xms128m", "-Xmx1024m"),
         // this enables custom javaOptions
         fork in run := true,
+        mappings in Universal ++= directory(baseDirectory.value / "sigar"),
+        bashScriptExtraDefines ++= Seq(
```
Comment (muuki88) [inline build.sbt]: smooth!

**Comment by muuki88** [comment]
Thanks a lot :)

File: build.sbt [hunk 3]
```diff
@@ -42,6 +50,11 @@ lazy val backend = (project in file("backend"))
...
+          """declare -r sigar_dir="$(realpath "${app_home}/../sigar")"""",
+          """addJava "-Djava.library.path=${sigar_dir}""""
+        ),
```
Comment (muuki88) [inline build.sbt]: With `1.0.0` you can specify `jvmOpts` directly with `jvmOpts in Universal`. However with this the access to the installation path will get trickier, so this solution is pretty nice.

**Comment by UriShohet** [comment]
Glad I could help and thanks for your comments!

**Pull request merged**